build-source.sh

```diff
@@ -17,9 +17,11 @@
 
 
 # Multi distro, set here to build master for multripple distros!
-MULTI_DIST="xenial bionic"
+MULTI_DIST="xenial focal"
 
-DIST="vivid xenial artful bionic focal"
+UBUNTU_DIST="xenial bionic focal groovy"
+DEBIAN_DIST="buster bullseye sid"
+DIST="$UBUNTU_DIST $DEBIAN_DIST"
```

Comment:
`*_DIST` -> `*_DISTS` and `DIST` -> `VALID_DISTS`. Current variable name clash with jenkins-debian-blue. If changed, we can avoid our custom `DIST_OVERRIDE` altogether (that requires un-patch `generate-git-snapshot` first).

Reply:
should not matter as we don't export the vars?

Reply:
Hmm, indeed it doesn't matter. Still, I would prefer not to create (or keep) confusion.

Reply:
Fixed